project_documentation/social_contract.txt

```diff
@@ -13,10 +13,52 @@ Social Contract
 
 * Unit testing strategy: TBD
 
+
+* we will have sprinte review and retrospective at 13.00-14.00 on every comming friday
+* Sprint reviews and retrospective meetings
+	consists of two parts
+	Sprint Review: ~15 min and
+	Sprint Retrospective:  (please prepare) ~40 min
+
+* During sprint review part we
+	Present what we as a team has accomplished, describe what user values we believe we provide.
+	get feedback from PO and present Stakeholders
+
+* During Sprint Retrospective part we discuss (please come prepared)
+	What has worked?
+	Where have we had problems?
+	What should we continue doing?
+	What should we improve?
+	What should we stop doing
+	
+	One person takes notes that will be upladed to the repo and linked to in the assignment report.
+
+* we will have sprinte planning at 14.00-15.00 on every comming friday, during which we 
+	* review the Product Backlog ~5 min
+
+	* Groom the top items
+		Groom: 
+			break down User Stories to tasks that can be SP estimated
+			ensure SP and tasks follow INVEST.
+
+	* take groomed items summing up to ~14 SP and add to sprint backlog
+
+	* preliminary distribute tasks to all members, to ensure all have something to work on.
+
+* we will have "daily" scrums on tuesday and thurdays from 9.00 to 9.15
+	during "daily" scrums each person presents in about 2 min.
+	* What have I done recently
+	* What do I plan to do next
+	* Am I blocked and if so how
+
+* we will start meetings at the specified time and late commers will be miss the beginning
+
+* When we don't have anything to do or we don't know what to do, we report that in Discord tin order to let other help explain or find meaningful tasks
+
 Tools:
 * version control: GitHub 
 * Sprintplanning: Trello
-* IDE: as preferred
+* IDE: as preferred, but  mainly Android Studio
 * Linter: TBD
 * communication and meetings: Discord
-* Language: likely candidates Python and/or Java. But depending on project.
\ No newline at end of file
+* Language: likely candidates Kotlin, Python and/or Java
```

Comment:
Didn't we decide of not having python?

Reply:
so Kotlin and Java then, right?

Reply:
Yes :)

Reply:
updated in commit 2
Im not sure how to make that reflect here though.

In the "Files changed" tab, i can choose "Changes from all commits" and see the removal of Python.
![image](https://user-images.githubusercontent.com/100037040/191094538-d471c6ca-4ac6-4c64-95c5-5adeb50d04b4.png)

Reply:
@antonsvaren If you are happy with my change in commit 2, please do Resolve conversation